saleor/graphql/vendor/mutations.py

```diff
@@ -0,0 +1,103 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ...core.permissions import VendorPermissions
+from ..core.utils import validate_slug_and_generate_if_needed
+from ...vendor import models
+from ...vendor.error_codes import VendorErrorCode
+from ..account.i18n import I18nMixin
+from ..core.mutations import ModelDeleteMutation, ModelMutation
+from ..core.types.common import VendorError
+from .types import VendorCreateInput, VendorWarehouseInput
+
+
```

Comment:
The mixin for ware house is missing here and it is required so that the create, update and delete classes inherit common methods like clean_input

Reply:
Since Update inherit create class so the clean_input present in the create class is inherited by the update class. (Reference: CategoryCreate)
For delete clean input is not required as of now because it removes the same row data. If we need to remove data from different model we may need clean_input in delete(confusion).